data/korath/korath weapons.txt

```diff
@@ -152,6 +236,82 @@ effect "fire-lance impact"
 	"random angle" 360
 	"velocity scale" 0.
 
+outfit "Blaze-Pike Turret"
+	category "Turrets"
+	cost 1970000
+	thumbnail "outfit/blaze-pike"
+	"mass" 57
+	"outfit space" -54
+	"weapon capacity" -54
+	"turret mounts" -1
+	"required crew" 1
+	weapon
+		sprite "projectile/blaze-pike"
+			"frame rate" 20
+		"hardpoint sprite" "hardpoint/blaze-pike"
+		sound "blaze-pike"
+		"hit effect" "blaze-pike hit"
+		"inaccuracy" 0.45
+		"turret turn" 2.8
+		"velocity" 450
+		"lifetime" 1
+		"reload" 1
+		"firing energy" 5
+		"firing heat" 4.5
+		"shield damage" 7.1
+		"hull damage" 3.1
+		"heat damage" 18
+		"burn damage" 0.45
+	description "The seeming reverence the Korath have for fire appears embodied in this weapon. This pair of Fire-Lances mounted together in a turret do not merely create twin lances of fire: Rather, they begin to form a gestalt - a single, longer, hotter, and almost hypnotic column of fire."
+
+effect "blaze-pike hit"
+	sprite "effect/blaze-pike hit"
+		"no repeat"
+		"frame rate" 30
+	"lifetime" 5
+	"random angle" 360
+	"random spin" 1
+	"random velocity" 5
+	"velocity scale" .005
+
+outfit "Korath Inferno"
```

Comment:
I think Raven said somewhere that she specifically wants to keep the 'Korath' on the Inferno because it is supposed to be something they take great pride in, or something along those lines.

Reply:
Yep, and the the rename PR adds a note in the description to call attention to that.